src/_langs/en/fundamentals/discovery-and-distribution/avoid-mixed-content/fixing-mixed-content.markdown

```diff
@@ -182,14 +192,16 @@ sections.
 CSP is useful for many things outside of its mixed content uses, you can find 
 more information about other CSP directives at the following resources:
 
-* Mozilla's intro to CSP: 
-  [https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Introducing\_Content\_Security\_Policy](https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Introducing_Content_Security_Policy) 
-* HTML5 Rocks intro to CSP: 
-  [http://www.html5rocks.com/en/tutorials/security/content-security-policy/](http://www.html5rocks.com/en/tutorials/security/content-security-policy/) 
-* CSP playground: [http://www.cspplayground.com/](http://www.cspplayground.com/) 
-* CSP version 2 spec: [http://www.w3.org/TR/CSP2/](http://www.w3.org/TR/CSP2/)
+* [Mozilla's intro to CSP](https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Introducing_Content_Security_Policy)
+* [HTML5 Rocks' intro to CSP](http://www.html5rocks.com/en/tutorials/security/content-security-policy/)
+* [CSP playground](http://www.cspplayground.com/)
+* [CSP version 2 spec](http://www.w3.org/TR/CSP2/)
+
+<!-- Span required to prevent rest of page from being indented.
+https://github.com/google/WebFundamentals/issues/1873 -->
+<span></span>
```

Comment:
@petele This is where I found issue #1873. <span></span> is my current workaround.